app/components/power-select/before-task-options.js

```diff
@@ -1,7 +1,25 @@
-import { get } from '@ember/object';
+import { get, set } from '@ember/object';
+import { on } from '@ember/object/evented';
 import BeforeOptionsComponent from 'ember-power-select/components/power-select/before-options';
+import {
+  EKMixin as EmberKeyboardMixin,
+  keyDown
+} from 'ember-keyboard';
+
+export default BeforeOptionsComponent.extend(EmberKeyboardMixin, {
+  init() {
+    this._super(...arguments);
+
+    set(this, 'keyboardActivated', true);
+  },
+
+  keydown: on(keyDown(), function(event, ekEvent) {
+    ekEvent.stopPropagation();
+    ekEvent.stopImmediatePropagation();
```

Comment:
I think this implicitly calls `stopPropagation`

Reply:
Is the reason for `stopImmediatePropagation` to prevent other events from firing on power-select?

Reply:
It doesn't. They're two separate things and explicated more in their docs.